Task6_GettingStartedwithAWS/Helper/SecretManagerHelper.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Amazon;
+using Amazon.SecretsManager;
+using Amazon.SecretsManager.Model;
+using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task6_AWS.Helper
+{
+    public class SecretsManagerHelper
+    {
+        private readonly IAmazonSecretsManager secretsManager;
```

Comment:
readonly properties should be named like "_secretsManager"

Reply:
corrected.